### ISSUE TITLE
Variable naming clarification

### ISSUE THREAD

**Issue by matheusmartin** [duolingo/halflife-regression#10]
I am trying to understand the Python implementation with a view to port it to another language, but can't make sense of some variables.

- `hlwt` at line [37](https://github.com/duolingo/halflife-regression/blob/0041df0dcd436bf1b4aa7a17a020d9c670db70d8/experiment.py#L37).
- `dlp_dw` at line [86](https://github.com/duolingo/halflife-regression/blob/0041df0dcd436bf1b4aa7a17a020d9c670db70d8/experiment.py#L86).
- `dlh_dw` at line [87](https://github.com/duolingo/halflife-regression/blob/0041df0dcd436bf1b4aa7a17a020d9c670db70d8/experiment.py#L87).
- `slp` at line [124](https://github.com/duolingo/halflife-regression/blob/0041df0dcd436bf1b4aa7a17a020d9c670db70d8/experiment.py#L124).
- `slh` at line [125](https://github.com/duolingo/halflife-regression/blob/0041df0dcd436bf1b4aa7a17a020d9c670db70d8/experiment.py#L125).

Could someone please explain what the acronyms for the above mean?

**Comment by theperiperi** [comment]
hlwt: Weight assigned to the half-life term in the loss function during training.
dlp_dw: Partial derivative of loss function w.r.t. recall probability weights for weight updates during training.
dlh_dw: Partial derivative of loss function w.r.t. half-life term weights for updates during training.
slp: Squared error between predicted and actual recall probabilities, part of loss function.
slh: Squared error between predicted and actual half-life values, part of loss function

This is what it seems like
